contracts/governance/extensions/GovernorStorage.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import "../Governor.sol";
+
+/**
+ * @dev Extension of {Governor} that implements storage of proposal details. This modules also provides primitives for
+ * the enumerability of proposals.
+ *
+ * Usecases for this module include:
+ * - UIs that expore the proposal state without relying on event indexing.
+ * - For some L2 chains where storage is compared to calldata, using a the proposalId version of queue and execute
```

Comment:
```suggestion
 * - Using only the proposalId as an argument in the {Governor-queue} and {Governor-execute} functions for L2 chains where storage is cheap compared to calldata.
```

Reply:
Just realized there is also {Governor-cancel}. Maybe enumerating them is too much ?